common/src/state/utils.rs

```diff
@@ -44,7 +44,16 @@ pub fn get_available_themes() -> Vec<Theme> {
     }
 
     themes.sort_by_key(|theme| theme.name.clone());
-    themes.dedup();
+
+    let default = Theme {
+        filename: "".into(),
+        name: "Default".into(),
+        styles: "".into(),
+    };
+
+    themes.push(default);
+
+    themes.dedup(); // Why are we deduping here? We check above to only add if the theme doesn't already exist
```

Comment:
Iirc this is done to remove any duplication (but do recall this being fixed else where but may be best to leave it in for now)